app/locales/en/translations.js

```diff
@@ -0,0 +1,233 @@
+export default {
+    global: {
+        share: `Share`,
```

Comment:
A bit of refactoring can be done to reduce duplication with string interpolation in the translation file--which is why I opted for template strings (AirBnB 6.4). Once we have a second translation, this will be easier to determine. Right now, it's a 1-for-1 substitution.

I was also originally planning for multi-line strings (didn't happen though).

Reducing escaping double or single quotes was another motivation. (AirBnB 6.6)